parser.js

```diff
@@ -64,7 +64,7 @@ module.exports = class Parser {
     let token = part.tokens[0]
     let node = new Comment()
     this.init(node, part)
-    node.source.end = { column: token[5], line: token[4] }
+    node.source.end = { offset: token[2] + token[1].length }
```

Comment:
Core CSS parser generates `offset` together with `line`/`column` for backward compatibility
https://github.com/postcss/postcss/blob/main/lib/parser.js#L356-L363

I think we should do it here as well